### PR TITLE
Add namespace for Concordium (ccd)

### PR DESCRIPTION
**Pull request by annenkov**

### CODE REVIEW

File: ccd/caip2.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the 
+ visible duplicate text guides, they will not appear and may be helpful to 
+ refer to if you edit it again. This is the suggested template for new EIPs.
+ Note that an EIP number will be assigned by an editor. When opening a pull
+ request to submit your EIP, please use an abbreviated title in the 
+ filename, `caipX.md`, all lowercase, no `-` between the CAIP and its 
+ number.-->
+This is the suggested template for new CAIPs.
```
Comment (bumblefudge) [inline ccd/caip2.md:18]:
```suggestion
```

File: ccd/caip2.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+# CAIP-2
+
+*For context, see the [CAIP-2][] specification.*
+
+<!-- Provide a simplified and layman-accessible explanation of the CAIP.-->
+
+This CAIP defines an identification schema for Concordium networks.
+The Concordium networks are identified by `ccd` namespace followed by the hash of the network's genesis block.
```
Comment (bumblefudge) [inline ccd/caip2.md:27]:
```suggestion
The Concordium networks are identified by the `ccd` namespace prefix, followed by a truncated hash of the given network's genesis block.
```

File: ccd/caip2.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+## Syntax
+
+<!-- Explain the actual algorithm or transformation needed to transform inputs into a
+conformant and unique CAIP deterministically.  Consider including a regular
+expression for validation as well, as some consumers or toolmakers may want to
+support this CAIP without a deep understanding of any specifications, devdocs,
+or improvement proposals on which this specification depends. -->
+
+Concordium's network ID is a hash of its genesis block in the `hex` encoding truncated to the first 32 characters.
```
Comment (bumblefudge) [inline ccd/caip2.md:42]:
```suggestion
Concordium's network ID is a hash of its genesis block in hexadecimal encoding, truncated to the first 32 characters.
```

File: ccd/caip2.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+<!-- TODO: add some details, e.g. something like pseudocode `truncate(SHA512(block_data), 32)` -->
+
+### Resolution Mechanics
+
+A hash of the genesis block can be obtained using the gRPC interface of a Concordium node (see [Concordium gRPC][]).
+The [GetConsensusInfo][] command provides data for constructing a network identifier.
```
Comment (bumblefudge) [inline ccd/caip2.md:49]:
```suggestion
The [GetConsensusInfo][] command provides data which includes the hash of that network's genesis block, which can be truncated to serve as the network identifier.
```

**Review by bumblefudge — APPROVED** [review]
Other than minor editorial, this LGTM, thanks!

**Comment by annenkov** [comment]
Thanks for your suggestions @bumblefudge! I applied them to the PR.

**Pull request merged**